.kokoro/dashboard.sh

```diff
@@ -20,9 +20,38 @@ scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 ## cd to the parent directory, i.e. the root of the git repo
 cd ${scriptDir}/..
 
+output="$scriptDir/dashboard/target/tmp/output.txt"
 ## Move into the dashboard directory
 cd dashboard/
 
 echo -e "\n******************** BUILDING THE DASHBOARD ********************"
 
 mvn --fail-at-end clean install
+INSTALL_RETURN_CODE=$?
+RETURN_CODE=${INSTALL_RETURN_CODE}
+
+LINE_COUNT=0
+
+case ${JOB_TYPE} in
+dependency-convergence-check)
+    mvn exec:java -Dexec.args="-f ../pom.xml --report"
+    CONVERGE_RETURN_CODE=$?
+    if [[ $INSTALL_RETURN_CODE -eq 0 ]]
+    then
+      while IFS= read -r line; do
+        msg "$line"
+        LINE_COUNT=$LINECOUNT+1
+      done < "$output"
```

Comment:
This might need to be `cat ${output}`. Maybe name it `outputFile` also as it's the path to the file not the output itself.